src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -380,23 +403,29 @@ private static bool BuildTargetFrameworkNode(PackageSpec packageSpec, KeyValuePa
             return true;
         }
 
-        private static NuGetFramework GetImports(JObject properties)
+        private static List<NuGetFramework> GetImports(JObject properties, PackageSpec packageSpec)
         {
-            NuGetFramework framework = null;
+            List<NuGetFramework> framework = null;
 
             var importsProperty = properties["imports"];
 
             if (importsProperty != null)
             {
-                var importFramework = NuGetFramework.Parse(importsProperty.ToString());
-
-                if (importFramework.IsPCL)
+                IEnumerable<string> importArray = new List<string>();
+                if (TryGetStringEnumerableFromJArray(importsProperty, out importArray))
                 {
-                    // PCLs are the only frameworks allowed here, other values will be ignored
-                    framework = importFramework;
+                    framework = importArray.Where(p => !string.IsNullOrEmpty(p)).Select(p => NuGetFramework.Parse(p)).ToList();
                 }
             }
 
+            if (framework != null && framework.Any(p => !p.IsSpecificFramework))
+            {
+                throw FileFormatException.Create(
+                           string.Format("The value of import frameworks in '{0}' is not a specific framework", PackageSpec.PackageSpecFileName),
```

Comment:
Give the invalid framework value. This should end with a .  Please add this to Strings.resx also.